traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -407,6 +408,9 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.1, http.MethodGet, `deliveryservices/{id}/urlkeys/?(\.json)?$`, deliveryservice.GetURLKeysByID, auth.PrivLevelReadOnly, Authenticated, nil},
 		{1.1, http.MethodGet, `riak/bucket/{bucket}/key/{key}/values/?(\.json)?$`, apiriak.GetBucketKey, auth.PrivLevelAdmin, Authenticated, nil},
 
+		//Pattern based consistent hashing endpoint
+		{1.5, http.MethodPost, `consistenthash/?$`, consistenthash.Post, auth.PrivLevelReadOnly, Authenticated, nil},
```

Comment:
if you do this make sure you change a couple of things:

1. https://github.com/apache/trafficcontrol/blob/master/traffic_portal/app/src/scripts/config.js#L26
2. add a section for 1.5 in https://github.com/apache/trafficcontrol/blob/master/traffic_ops/app/lib/TrafficOpsRoutes.pm#L61